src/modules/profile/Profile.tsx

```diff
@@ -0,0 +1,210 @@
+`;
+
+const StyledAccountButtons = styled('button')`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  border: none;
+  border-bottom: 2px solid #f6f6f6;
+  background-color: transparent;
+  cursor: pointer;
+  padding: 20px;
+`;
+const StyledAccountButtonsArea = styled('div')`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 52px;
+  height: 52px;
+  border-radius: 16px;
+  background-color: #eee5ff;
+`;
+const StyledLogoutButton = styled('button')`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+  padding: 20px;
+`;
+const StyledAccountLogoutButtonArea = styled('div')`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 52px;
+  height: 52px;
+  border-radius: 16px;
+  background-color: #ffe2e4;
+`;
+
+export const Profile = () => {
+  const { users } = useAppContext();
+  return (
+    <StyledDivProfile>
+      {users.map((user) => (
+        <StyledDivUserDetailsSection key={user.id}>
+          <StyledDivProfilePicture>
+            {user.profilePicture ? (
+              <img
+                style={{
+                  width: '100%',
+                  height: '100%',
+                  borderRadius: '50%',
+                  objectFit: 'cover',
+                }}
+                src={user.profilePicture}
+                alt={`${user.name} profile`}
+              />
+            ) : (
+              <PermIdentityIcon
+                style={{
+                  width: '100%',
+                  height: '100%',
+                  borderRadius: '50%',
+                  background: 'lightGray',
+                  objectFit: 'cover',
+                }}
+              />
+            )}
+          </StyledDivProfilePicture>
+
+          <StyledDivUserInfo>
+            <Typography
+              style={{ fontWeight: '500', fontSize: '14px', color: '91919F' }}
+            >
+              {user.username}
+            </Typography>
+            <Typography style={{ fontWeight: '600', fontSize: '24px' }}>
+              {user.name} {user.surname}
+            </Typography>
+          </StyledDivUserInfo>
+
+          <StyledDivEditProfileButton>
+            <EditOutlinedIcon style={{ color: 'black', fontSize: '30px' }} />
+          </StyledDivEditProfileButton>
+        </StyledDivUserDetailsSection>
+      ))}
+      <StyledDivAccountButtonGroup>
+        <StyledAccountButtons>
+          <StyledAccountButtonsArea>
+            <AccountBalanceWalletIcon
+              style={{
+                width: '32px',
+                height: '32px',
+                color: colors.violet[100],
+              }}
+            />
+          </StyledAccountButtonsArea>
+          <Typography
+            style={{ fontWeight: '500', marginLeft: '10px', fontSize: '16px' }}
+          >
+            Account
+          </Typography>
+        </StyledAccountButtons>
+        <StyledAccountButtons>
```

Comment:
This could be extracted into a separate component. Make sure it takes icon, title, and props for color and bottom-border so it can be reused for all list items.